### ISSUE TITLE
[Bug]: 3.13.4 Infinite Reboot Loop on Windows 10

### ISSUE THREAD

**Issue by github-cli** [nextcloud/desktop#7214]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

I have had an infinite reboot loop happening so I kept doing a system restore (only way I could get the system to boot again) and noticed that this issue kept coming back once I upgraded the Desktop client from 3.13.3 to 3.13.4 (server version 29.0.7)
The reboot would happen even on the login screen before logging in if I did not login quick enough.

I have now installed 3.14 and it seems to not happen anymore but the main panel does not come up until I close and open the application again (which has been reported by others already) but even then, the main panel is unusable, the font is always white, a lot of the symbols are white as well while the background is a light gray.

### Steps to reproduce

not sure if it is reproducible but I was running 3.13.3 on Windows 10 22H2 and upgraded to 3.13.4 and rebooted after which it kept entering the reboot loop
this is reproducible on my PC at least

### Expected behavior

System behaves normal

### Which files are affected by this bug

all

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10 22H2

### Package

Official Windows MSI

### Nextcloud Server version

29.07.1

### Nextcloud Desktop Client version

3.13.4

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [X] LDAP/ Active Directory
- [X] SSO - SAML
- [ ] Other

**Comment by mgallien** [comment]
@github-cli I am closing it as you said that 3.14.0 is not affected.
I would also suggest that next time, you would reach out for help from the community via the forum https://help.nextcloud.com
I would suggest that you keep investigating why your computer was rebooting in a loop

**Comment by github-cli** [comment]
@mgallien I thought bugs would be reported on github and help with e.g. configurations would be on the forum?
If you meant during the troubleshooting phase... for one I did not know it was because of Nextcloud and once I knew, there was no reason to ask anymore.
About the troubleshooting: I did it as far as my knowledge reasonably takes me to, the reason was the upgrade of the nextcloud desktop app. why exactly I do not know and also do not know how to properly troubleshoot but I will gladly provide any information that might help figure this out.

For now I have downgraded to 3.12.7 as 3.14 has other issues that make it unusable.

Just for the future, problems I encounter in one version but dont see in a subsequent version, even if as severe as this one, should not be reported?
